modules/dop/services/pipeline/pipeline.go

```diff
@@ -368,7 +368,8 @@ func (p *Pipeline) ConvertPipelineToV2(pv1 *apistructs.PipelineCreateRequest) (*
 	pv2.ConfigManageNamespaces = append(pv2.ConfigManageNamespaces, ns...)
 
 	// label
-	labels[apistructs.LabelDiceWorkspace] = workspace
+	// orchestrator judge the branch to get workspace,so dop don't need add workspace to labels
+	//labels[apistructs.LabelDiceWorkspace] = workspace
 	labels[apistructs.LabelBranch] = pv1.Branch
```

Comment:
does branch also useless?

Reply:
branch is useless for `dice` action, but it may affect other actions like `git-checkout`

Reply:
i see